### PR TITLE
Review Code Pattern - Maintenance

### PR DESCRIPTION
**Pull request by yil532**
The related issue is [here](https://github.ibm.com/CODAIT/Model-Asset-Exchange/issues/340).

### CODE REVIEW

File: notebooks/opioid-prescription-prediction.ipynb [hunk 1]
```diff
@@ -671,300 +754,102 @@
...
...OpnHjxsTFxZGZmUnHjh1Nnqm8f3nrZ599xgcffMCKFSto2bIld+7c4ddffzU57+7du9m4cSO2trZMmzaN+fPn88knnzx0fb1ez9ixY2nVqhUbNmzg5s2bTJw48aHltGPHjqV58+aoVCrWrVtHUFAQBw4cwM7OrtjXfDQHwKUY44UQQgghhBBllRSWxeTt7U3fvn0JCQkhJiaGYcOGGWcmCxMVFcWYMWOMM4bW1tZ4eXmZjBk/fjxVq1YFYPDgwUyePLnQc505c4bLly+zY8cOLC0tqVatGuPGjSMwMNA4RqfTodPpjK+DgoLYvHkz8fHxdO7cudjXFEII8eJSq9VoNMpqx6DRqO77qazYXzSSK2WQPClDQZ7MSQrLYnJ0dDTOGB4+fJjg4GDu3r37yCY8iYmJ1K5d+5HnU6lUVKtWzfjaysqKO3fukJeX91DBevXqVbRaLVZWVsZj1atXNxmTnJzM0qVLOXPmDLdv30alUpGZmcn169ef6ppCCCFeXFqtldn3RXtaWq21uUMQRSS5UgbJk3gSKSyfgo+PD2PGjKF3797Y29uTlJT0yLHVq1fnzz//fCbXdXR0JD09naysLKyt8//lfvDac+bMoVKlSmzbts249LVFixZF2v5ECCGEuF96ejZpaZnmDqNYNBoVWq016elZ6PXy/77STHKlDJInZSjIkzlJYfkUOnToQGRkJHXq1DEee1Th5uvry6pVq3B3d6dly5ZkZ2fz66+/0qJFi8d+rjBNmzbF1dWVpUuXMnPmTNLT01m3bp3JmIyMDJycnLC1tSUrK4uIiAiysrJMxjybIjP1GZxDCCFE6ZVKXl4eer3SOuDmr7zR6w0KjP1FI7lSBsmTMph/1aEUlkX04FLXgmcmH/V+gaFDh6LRaFi0aBFJSUlYWVnRr18/Y2FZ1P0uIb950Nq1awkNDaVt27Y4OzszfPhwfvrpJ8qXLw/kz1jOmzePli1bUqVKFfz9/XFycipSrMURFxfyQrXcV6IXcWsEpZJcKcOLmCcPjwbmDkEIIYRCqAyyRlLRDh06xNtvv83PP//83K+dlpYpv7kqxTQaNXZ2NpInBZBcKYPkSRkkT8ohuVIGyZMyFOTJnMw/ZyqK5aeffjI+s3nx4kVWr15N//79zRuUEEIIIYQQ4oUmS2GfoVmzZqHX61myZEmJXSM1NZWpU6dy48YNtFo
```
Comment (stevemar) [inline notebooks/opioid-prescription-prediction.ipynb]: you'll want to remove these api keys and tokens

**Review by stevemar — COMMENTED** [review]
can you also check in the PDF that we found online?

File: notebooks/opioid-prescription-prediction.ipynb [hunk 2]
```diff
@@ -1515,27 +1416,25 @@
    "source": [
     " - \"Quick and Dirty\" approach from Kaggle (https://www.kaggle.com/jiashenliu/quick-and-dirty-attempt-on-voting-classifier)\n",
     " - \"Detecting Frequent Opioid Prescription\" (https://inclass.kaggle.com/apryor6/detecting-frequent-opioid-prescription)\n",
-    " - Indiana University: \"Opiate prescription analysis using machine learning\" (http://cgi.soic.indiana.edu/~arunsank/AML_report.pdf)"
+    " - Indiana University: \"Opiate prescription analysis using machine learning\" (https://github.com/anirudhkm/opiate-prescription-analysis/blob/master/AML_report.pdf)"
```
Comment (yil532) [inline notebooks/opioid-prescription-prediction.ipynb]: The pdf is here @stevemar

File: notebooks/opioid-prescription-prediction.ipynb (hunk 1 above)
Comment (yil532) [reply notebooks/opioid-prescription-prediction.ipynb]: Sorry, forgot about this. Should I clear out all of the output of the notebook as well then?

**Comment by stevemar** [comment]
LGTM

**Pull request merged**